data/tickets.yml

```diff
@@ -1,38 +1,38 @@
-live: false
+live: true
 data:
-  - 
+  -
     name: Early Bird Ticket
-    details: Sale ends by 22 August, 2020 11:59 PM (Asia/Kolkata) OR completion of 300 tickets
-    price: 149
+    details: Sale ends by 22 August, 2021 11:30 PM (Asia/Kolkata) OR completion of 100 tickets
+    price: 450
     category: conference
     link: ""
 
-  - 
+  -
     name: Regular Ticket
-    details: Sale ends by 15 September, 2020 11:59 PM (Asia/Kolkata) OR completion of 2000 tickets
-    price: 199
+    details: Sale ends by 15 September, 2021 11:30 PM (Asia/Kolkata) OR completion of 800 tickets
+    price: 600
     category: conference
     link: ""
```

Comment:
Please add links to the townscripts ticket here and other tickets before release.

Reply:
Sure, let me fix this.